src/HID-APIs/D3MouseAPI.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <Arduino.h>
+#include "HID-Settings.h"
+
+typedef union {
```

Comment:
Add packed attribute so the compiler does not add pad bytes between elements.

`typedef union ATTRIBUTE_PACKED {`

Reply:
```suggestion
typedef union ATTRIBUTE_PACKED {
```